### PR TITLE
feat: bp activity fead code clean up

### PR DESCRIPTION
**Pull request by roger-in-kiva**
- bp activity feed code clean up
- packages updated

### CODE REVIEW

File: src/components/BorrowerProfile/LendCta.vue [hunk 1]
```diff
@@ -575,11 +549,7 @@ export default {
 		},
 	},
 	methods: {
-		async addToBasket(lendAmount = 0) {
-			if (lendAmount) {
-				this.$kvTrackEvent('Borrower profile', 'click', 'loan-activities-lend', this.loan?.id, lendAmount);
```
Comment (dyersituations) [inline src/components/BorrowerProfile/LendCta.vue:580]: Are we sure this event was specific to activity feed? Was it part of those PRs?

Comment (roger-in-kiva) [reply src/components/BorrowerProfile/LendCta.vue:580]: Yeah, we added for this case only.

File: src/pages/BorrowerProfile/BorrowerProfile.vue [hunk 2]
```diff
@@ -611,22 +609,6 @@ export default {
...
-			ACTIVITY_FEED_EXP,
-			'EXP-ACK-1037-MAR2024',
-		);
-		if (activityFeedExpData?.version === 'b') {
-			const response = await this.apollo.query({
-				query: loanActivitiesQuery,
-				variables: { loanId: this.loanId }
-			});
-
-			this.activities = response?.data ?? null;
```
Comment (dyersituations) [inline src/pages/BorrowerProfile/BorrowerProfile.vue:627]: Looks like this data prop needs to be removed as well?

Comment (dyersituations) [reply src/pages/BorrowerProfile/BorrowerProfile.vue:627]: It can also be removed from `data`

Comment (roger-in-kiva) [reply src/pages/BorrowerProfile/BorrowerProfile.vue:627]: thanks

File: src/components/BorrowerProfile/LendCta.vue [hunk 3]
```diff
@@ -591,7 +561,6 @@ export default {
 
 			this.isAdding = true;
 			this.errorMsg = '';
-			this.selectedOption = Number(lendAmount) || this.selectedOption;
```
Comment (dyersituations) [inline src/components/BorrowerProfile/LendCta.vue:594]: Why was this removed? It looks like it's still being used.

Comment (roger-in-kiva) [reply src/components/BorrowerProfile/LendCta.vue:594]: We added this to consider lendAmount different than 0, no more used, also the errorMsg.

Comment (roger-in-kiva) [reply src/components/BorrowerProfile/LendCta.vue:594]: `selectedOption` is coming from the dropdown...

Comment (roger-in-kiva) [reply src/components/BorrowerProfile/LendCta.vue:594]: https://github.com/kiva/ui/commit/645c6bac408a54319adb8c1613bdced17154af55#diff-09ca3cc753df5d76e7f1404c47aa4ee2f07ce9ced29b352f4739cce2c1db3168L588

**Review by dyersituations — APPROVED** [review]
